### PR TITLE
[ Fixed Issue #667 ] Add Google and Unsplash terms links to experimental/desktop_photo_search

### PR DESCRIPTION
**Pull request by SatYu26**
Fixed [Issue #667](https://github.com/flutter/samples/issues/667)

I have made few changes according to the requirement in this issue.
As of now the dialog looks like this:

Screenshot 2021-01-28 025434

I have added sentences according to the information available to me but if you let me know the specific details of the statements i will make those changes quickly.

Also i have added the access point for this dialog in About menu inside the menubar, As you can see in this picture.

I will can make more changes also if required by you.

### CODE REVIEW

**Comment by SatYu26** [comment]
@domesticmouse  Sir i solved the issue but there seems to be some checks that have not cleared.
I dont know what might be the issue for these checks not passing. is it possible if you guide me through it.
P.s. I am on master channel when i created these changes.

Screenshot 2021-01-28 021329

File: experimental/desktop_photo_search/lib/main.dart [hunk 1]
```diff
@@ -19,6 +19,7 @@ import 'src/widgets/photo_details.dart';
 import 'src/widgets/photo_search_dialog.dart';
 import 'src/widgets/split.dart';
 import 'unsplash_access_key.dart';
+import 'src/widgets/about_dialog.dart';
```
Comment (domesticmouse) [inline experimental/desktop_photo_search/lib/main.dart:22]: Can you please re-order these imports to be alphabetical? The following error is showing up in the CI

```
info • Sort directive sections alphabetically • lib/main.dart:22:1 • directives_ordering
```

**Review by domesticmouse — CHANGES_REQUESTED** [review]
First up, thanks for the PR!

Please review the CI pipeline and correct the issue(s) that it notes. We can't land a PR that is red. Thanks for your assistance =)

brett

File: experimental/desktop_photo_search/lib/main.dart (hunk 1 above)
Comment (SatYu26) [reply experimental/desktop_photo_search/lib/main.dart]: Sure sir.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
Thank you!

**Comment by SatYu26** [comment]
@domesticmouse Thank you sir.

**Pull request merged**